app/models/team/Organization.scala

```diff
@@ -22,20 +19,21 @@ case class OrganizationSQL(
                             displayName: String,
                             created: Long = System.currentTimeMillis(),
                             isDeleted: Boolean = false
-                          )
+                          ) {
 
+  def organizationTeamId(implicit ctx: DBAccessContext): Fox[ObjectId] =
+    OrganizationSQLDAO.findOrganizationTeamId(_id)
 
-object OrganizationSQL {
-  def fromOrganization(o: Organization)(implicit ctx: DBAccessContext): Fox[OrganizationSQL] = {
-    Fox.successful(
-      OrganizationSQL(
-        ObjectId.fromBsonId(o._id),
-        o.name,
-        o.additionalInformation,
-        o.logoUrl,
-        o.displayName
-      )
-    )
+  def teamIds(implicit ctx: DBAccessContext): Fox[List[ObjectId]] =
+    TeamSQLDAO.findAllIdsByOrganization(_id)
+
+  def publicWrites(implicit ctx: DBAccessContext): Fox[JsObject] = {
+    Fox.successful(Json.obj(
+      "id" -> _id.toString,
+      "name" -> name,
+      "additionalInformation" -> additionalInformation,
+      "displayname" -> displayName
```

Comment:
note to self: @philippotto spoke about renaming this to `displayName` in the json as well, needs to be done here, too